### PR TITLE
Filter building time stats

### PR DESCRIPTION
**Pull request by geohacker**
cc @kamicut

### CODE REVIEW

File: validators/filterResidentialBuildings/README.md [hunk 1]
```diff
@@ -1,6 +1,20 @@
...
 
-Filter buildings with tag `building=residential`. Returns a line delimeted feature collection to stdout.
+Filter buildings with tag `building=residential`. Returns a line delimeted feature collection to stdout. The linter will return a JSON bin for edit recency statistics with the following format:
+
+```
+{
+  'YYYYM': number of buildings,
+  '20138': 75,
+  '20139': 416,
+  '20140': 363,
```
Comment (kamicut) [inline validators/filterResidentialBuildings/README.md]: @geohacker is the month zero indexed? on month 10,11,12 shouldn't the format be "YYYYMM" ?

File: validators/filterResidentialBuildings/map.js [hunk 2]
```diff
@@ -1,15 +1,29 @@
...
     var val = layer.features[i];
     if (val.properties.hasOwnProperty('building') && val.properties['building'] === 'residential') {
       val.properties._osmlint = osmlint;
       result.push(val);
+
+      // populate timeBucket( for edit recency analysis
+      var timestamp = moment(val.properties['@timestamp'] * 1000);
+      var year = timestamp.year();
+      var month = timestamp.month();
+      var key = String(year) + String(month);
```
Comment (kamicut) [inline validators/filterResidentialBuildings/map.js]: This should change, see above comment

File: validators/filterResidentialBuildings/map.js [hunk 3]
```diff
@@ -1,15 +1,29 @@
 'use strict';
 var turf = require('@turf/turf');
+var moment = require('moment');
```
Comment (kamicut) [inline validators/filterResidentialBuildings/map.js:3]: Have you checked 'date-fns'? I found it to be smaller and have better tools. moment works great too, just letting you know about an alternative.

**Review by kamicut — CHANGES_REQUESTED** [review]
I think this needs a minor change.

File: validators/filterResidentialBuildings/map.js (hunk 2 above)
Comment (geohacker) [reply validators/filterResidentialBuildings/map.js]: Oh haha, can't believe I missed that. Thank you for catching @kamicut!

File: validators/filterResidentialBuildings/README.md (hunk 1 above)
Comment (geohacker) [reply validators/filterResidentialBuildings/README.md]: You're right. Moment uses zero indexed months. I'd say we keep it this way. https://momentjs.com/docs/#/get-set/month/

**Pull request merged**